src/main/java/edu/ucsb/nceas/mdqengine/scheduler/MonitorJob.java

```diff
@@ -0,0 +1,300 @@
+package edu.ucsb.nceas.mdqengine.scheduler;
+
+import edu.ucsb.nceas.mdqengine.exception.MetadigStoreException;
+import edu.ucsb.nceas.mdqengine.model.Run;
+import edu.ucsb.nceas.mdqengine.MDQconfig;
+import edu.ucsb.nceas.mdqengine.DataONE;
+import edu.ucsb.nceas.mdqengine.store.DatabaseStore;
+import edu.ucsb.nceas.mdqengine.store.MDQStore;
+import edu.ucsb.nceas.mdqengine.Controller;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+
+import org.dataone.client.rest.HttpMultipartRestClient;
+import org.dataone.client.rest.MultipartRestClient;
+import org.dataone.client.v2.impl.MultipartCNode;
+import org.dataone.client.v2.impl.MultipartMNode;
+import org.dataone.exceptions.MarshallingException;
+import org.dataone.service.types.v1.Session;
+import org.dataone.service.exceptions.NotAuthorized;
+import org.dataone.service.exceptions.*;
+import org.dataone.service.types.v1.*;
+import org.dataone.service.types.v2.SystemMetadata;
+import org.dataone.service.util.*;
+
+import org.quartz.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import java.io.InputStream;
+
+import java.io.ByteArrayOutputStream;
+import java.io.ByteArrayInputStream;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+/**
+ * A Quartz job implementation for monitoring for metadata quality requests
+ * 'stuck' with a status of 'processing' in the persistent store for more than
+ * 24 hours.
```

Comment:
okay I'll explain more. the stuck state would arise if the worker is killed sometime while it is processing the report. because of the pre-emptive ack we have completely eliminated the RMQ timeout. the worker can take as long as it wants to finish a job, essentially